release/api/v1alpha1/bundle_types.go

```diff
@@ -226,6 +227,11 @@ type FluxBundle struct {
 	NotificationController Image  `json:"notificationController"`
 }
 
+type PackageBundle struct {
+	Version           string `json:"version,omitempty"`
+	PackageController Image  `json:"packageController"`
```

Comment:
`Controller` or `Manager` to avoid stutters when referencing , i.e., `PackageController.PackageController`